.flake8

```diff
@@ -4,4 +4,7 @@ exclude =
     *migrations*
     ./config/settings.py
     ./team_production_system/management/commands/add_superuser.py
-
+max-line-length = 80
```

Comment:
There might be a conflict with this and black auto linting. I believe you have black set to max line length of 88.

Reply:
The bugbear plugin allows for a 10% safety net, so the max-line-length is effectively 88. This is implemented by lines 8 and 9. I've added comments to address this.